js/base/Exchange.js

```diff
@@ -92,6 +92,7 @@ module.exports = class Exchange {
                 'fetchDeposit': undefined,
                 'fetchDepositAddress': undefined,
                 'fetchDepositAddresses': undefined,
+                'fetchDepositAddressesByNetwork': undefined,
```

Comment:
How did it get removed? 

- https://github.com/ccxt/ccxt/blob/master/js/huobi.js#L2579

Reply:
Nevermind, my bad. Wrong file.